### ISSUE TITLE
Publish multi arch manifest

### ISSUE THREAD

**Issue by christoffer-eide** [viaduct-ai/kustomize-sops#167]
Support for building multi arch was added a while back in https://github.com/viaduct-ai/kustomize-sops/pull/102

It would be of great help if a multi arch manifest could be published to [docker hub](https://hub.docker.com/r/viaductoss/ksops/tags). The tag `v3.1.0` is built for `linux/amd64`, and the tag `v3.1.0-arm64` is built for `linux/arm64`. 

This makes it cumbersome to build a multi arch argocd image following the [instructions in the readme](https://github.com/viaduct-ai/kustomize-sops/blob/master/README-legacy.md#custom-argo-cd-w-ksops-dockerfile).

There a several options here.

- Publish an additional `v3.1.0-amd64` tag. This would allow us to use the [`$TARGETARCH`](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope) ARG when building the image:
```
ARG ARGO_CD_VERSION="v1.7.7"
ARG KSOPS_VERSION="v3.1.0"

FROM viaductoss/ksops:$KSOPS_VERSION-$TARGETARCH as ksops-builder
FROM argoproj/argocd:$ARGO_CD_VERSION
```

- Publish a multi arch manifest when building ksops:
```
docker buildx build --platform=linux/amd64,linux/arm64 --tag viaductoss/ksops:v3.1.0-multiarch  .
```

- Publish a multi arch manifest "after the fact":
```
docker buildx imagetools create 
--tag viaductoss/ksops:v3.1.0-multiarch
viaductoss/ksops:v3.1.0
viaductoss/ksops:v3.1.0-arm64
```

**Comment by devstein** [comment]
Hi @christoffer-eide thanks for making an issue! Think this is a great idea. 

Are there any downsides you can think of for using `buildx` for all images and publishing a single multi-arch manifest under the `vX.X.X` tag (dropping the `-arm64` tags)? For example,

```bash
docker buildx build --platform=linux/amd64,linux/arm64 --tag viaductoss/ksops:v3.1.0 .
```

**Comment by christoffer-eide** [comment]
@devstein Thanks for the prompt feedback 👍 

If you start to publish a multi arch image using only the `v3.1.0` tag, it should work out of the box. Most projects that build multi arch uses this approach.

There might be users that rely on tags such as `v3-arm64`, and if you stop publishing those tags they will not "get" the latest 3.x.x version.

**Comment by devstein** [comment]
@christoffer-eide I released [`v4.0.1`](https://github.com/viaduct-ai/kustomize-sops/releases/tag/v4.0.1) and `v3.1.1` which have [multi-arch manifests](https://hub.docker.com/r/viaductoss/ksops/tags). All releases will follow this format moving forward. 



Thanks for raising this!
